### PR TITLE
setup the eslint and the prettier, connecting eslint with prettier #40

### PR DESCRIPTION
**Pull request by karmelyoei**

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -30,5 +30,11 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-plugin-import": "^2.20.2",
```
Comment (RandInaim) [inline client/package.json:38]: please install  `eslint-config-prettier` 
to turn off all ESLint rules that are unnecessary or might conflict with Prettier rules.

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+	"env": {
+		"browser": true,
+		"commonjs": true,
+		"es6": true,
+		"node": true
+	},
+	"extends": ["airbnb-base"],
```
Comment (RandInaim) [inline .eslintrc.json]: you need to extends prettier also 
should be like this "extends": [  "prettier",  "airbnb-base"]

File: client/package.json (hunk 1 above)
Comment (karmelyoei) [reply client/package.json:38]: I installed it in the dependencies is that ok or should I switch it  ??

File: .eslintrc.json (hunk 2 above)
Comment (karmelyoei) [reply .eslintrc.json]: Done

File: client/package.json (hunk 1 above)
Comment (karmelyoei) [reply client/package.json:38]: I have changed it from dependencies into dev ...Done

**Pull request merged**